.stylelintrc

```diff
@@ -1,13 +1,10 @@
 {
   "enabled": true,
-  "extends": [
-    "stylelint-config-recommended-scss"
-  ],
-  "plugins": [
-    "stylelint-scss"
-  ],
+  "extends": ["stylelint-config-recommended-scss"],
+  "plugins": ["stylelint-scss", "stylelint-use-logical"],
```

Comment:
`stylelint-use-logical` looks like a really useful plugin, one to keep an eye on but it doesn't look particularly active or highly used. We've got much worse dependencies than this in the projects site we need to sort but might be worth checking to see if other options are around